PSWebServiceLibrary.php

```diff
@@ -215,7 +215,7 @@ protected function executeRequest($url, $curl_params = array())
         curl_close($session);
         if ($this->debug) {
             if ($curl_params[CURLOPT_CUSTOMREQUEST] == 'PUT' || $curl_params[CURLOPT_CUSTOMREQUEST] == 'POST') {
-                $this->printDebug('XML SENT', urldecode($curl_params[CURLOPT_POSTFIELDS]));
+                $this->printDebug('XML SENT', urldecode(var_export($curl_params[CURLOPT_POSTFIELDS], true)));
```

Comment:
Why not use `array_reduce` ?

Reply:
It may be better, but this is just a basic way to avoid critical errors.